### PR TITLE
chap via env

### PR DESCRIPTION
**Pull request by sijeesh**

### CODE REVIEW

**Review by dileepds — APPROVED** [review]
LGTM

File: pkg/flavor/kubernetes/flavor.go [hunk 1]
```diff
@@ -45,6 +45,8 @@ const (
 	chapSecretNamespaceKey        = "chapSecretNamespace"
 	chapUserKey                   = "chapUser"
 	chapPasswordKey               = "chapPassword"
+	chapSecretNameEnvKey          = "CHAP_SECRET_NAME"
+	chapSecretNamespaceEnvKey     = "CHAP_SECRET_NAMESPACE"
 	chapUserValidationPattern     = "^[a-zA-Z0-9][a-zA-Z0-9\\-:.]{0,63}$"
 	chapPasswordValidationPattern = "^$|^[a-zA-Z0-9+_)(*^%$#@!]{12,16}$"
```
Comment (datamattsson) [inline pkg/flavor/kubernetes/flavor.go:51]: This regex doesn't contain the `-` either (as in co-deployments)

Comment (sijeesh) [reply pkg/flavor/kubernetes/flavor.go]: Updated

File: pkg/flavor/types.go [hunk 2]
```diff
@@ -39,5 +39,6 @@ type Flavor interface {
 	GetOrchestratorVersion() (*version.Info, error)
 	MonitorPod(podLabelkey, podLabelvalue string) error
 	GetGroupSnapshotNameFromSnapshotName(snapshotName string) (string, error)
+  ListVolumeAttachments() (*storage_v1.VolumeAttachmentList, error)
```
Comment (datamattsson) [inline pkg/flavor/types.go:42]: Where did this come from?

Comment (sijeesh) [reply pkg/flavor/types.go]: Fixed indentation

**Review by datamattsson — APPROVED** [review]
LGTM

**Pull request merged**